src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -149,6 +154,11 @@ const SignOrExecuteForm = ({
     setAdvancedParams(data)
   }
 
+  const hangleExecuteCheckboxChange = (checked: boolean) => {
```

Comment:
There is a small typo here

```suggestion
  const handleExecuteCheckboxChange = (checked: boolean) => {
```